Makefile

```diff
@@ -107,3 +107,22 @@ else
 	wget https://s3.amazonaws.com/rebar3/rebar3
 	chmod a+x rebar3
 endif
+
+travis-publish:
+	@echo Create directories
+	mkdir -p ~/.hex
+	mkdir -p ~/.config/rebar3
+
+	@echo Decrypt secrets
+	@openssl aes-256-cbc -K $encrypted_9abc06b32f03_key -iv $encrypted_9abc06b32f03_iv -in hex.config.enc -out ~/.hex/hex.config -d
+
+	@echo Create global config
+	echo '{plugins, [rebar3_hex]}.' > ~/.config/rebar3/rebar.config
```

Comment:
I'm actually putting this into the _global_ rebar3 config, because I don't want ercloud's dependencies to include it. I may be overthinking things, though; I'm not sure what rebar actually does if there are plugin dependencies listed in a Hex dependency.